tracer/src/Datadog.Trace/HttpHeaderNames.cs

```diff
@@ -50,7 +50,14 @@ public static class HttpHeaderNames
 
         /// <summary>
         /// ID of a span.
+        /// Used in a serverless context only.
         /// </summary>
         public const string SpanId = "x-datadog-span-id";
+
+        /// <summary>
+        /// If header is set to "true", the extension will know that the current invocation has failed
+        /// Used in a serverless context only.
+        /// </summary>
+        public const string InvocationError = "x-datadog-invocation-error";
```

Comment:
If these are used in a serverless context only, they should probably be internal (we don't expect/want users to use them)

Reply:
Good catch! Same for `HttpHeaderNames.SpanId` added previously in #2233, not released yet.

Reply:
✅